### PR TITLE
Updated button component

### PR DESCRIPTION
**Pull request by hanna446**
image

### CODE REVIEW

**Comment by jusk02** [comment]
Please resolve your code conflicts

**Comment by jhonatanrojasbastidas** [comment]
recheck te media query you have more code that you need.

**Comment by camigomez35** [comment]
Conflicts!

File: pages/buttons.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (jusk02) [inline pages/buttons.html]: This is a testing file, and should not be included in the PR

File: css/style.css [hunk 2]
```diff
@@ -1,5 +1,105 @@
...
+
+  .color--secondary:hover {
+    background-color: #003c8f;
+  }
+
+  .color--secondary-dark:hover {
+    background-color: #1565c0;
+  }
+
+  .color--primary-light:hover{
```
Comment (jusk02) [inline css/style.css]: please add a space between the selector and the bracket

File: css/style.css [hunk 3]
```diff
@@ -1,5 +1,105 @@
...
+  font-size: 16px;
+}
+
+.button--lg {
+  min-width: 320px;
+  height: 60px;
+  font-size: 22px;
+}
+
+.color--primary {
```
Comment (camigomezdev) [inline css/style.css]: All color classes for this component should be called `button--{color}`.

**Comment by jhonatanrojasbastidas** [comment]
@jusk02 @camigomez35 @josemariogutierrez  somebody can answer my question about her media query?

**Comment by jusk02** [comment]
> @jusk02 @camigomez35 @josemariogutierrez somebody can answer my question about her media query?

you are right, some rules are set with the same value on both sides of the media query.

@hanna446 please make sure you are not repeating rules on desktop if they are already set in mobile with the same value.

File: css/style.css (hunk 3 above)
Comment (christian-cg) [reply css/style.css]: does `max-width: 100%` actually works?

File: css/style.css [hunk 4]
```diff
@@ -56,74 +156,12 @@ body {
...
-
-.button--secondary-dark {
-	color: #ffffff;
-	background-color: #003c8f;
-}
-
-.button--danger {
-	color: #ffffff;
-	background-color: #ce4141;
-}
```
Comment (christian-cg) [inline css/style.css:184]: :(

File: css/style.css (hunk 3 above)
Comment (jusk02) [reply css/style.css]: @christiang6 Yes, this would stop the button from growing over the width of its container element.

File: css/style.css [hunk 5]
```diff
@@ -105,94 +159,61 @@ video {
...
+  .button--color--primary:hover {
+    background-color: #0086c3;
+  }
+  
+  .button--color--primary-dark:hover {
+    background-color: #29b6f6;
+  }
 
-.button--secondary-light {
```
Comment (camigomezdev) [inline css/style.css:172]: this is the correct naming convention for the secondary color of the button.
please check your class names.

**Comment by jhonatanrojasbastidas** [comment]
you are not answer the question we write for you.

**Comment by hanna446** [comment]
> you are not answer the question we write for you.

Yes Jhonatan I'm sorry.
I already deleted the unnecessary code

**Comment by jhonatanrojasbastidas** [comment]
look good.

**Review by jhonatanrojasbastidas — APPROVED** [review]
ready for merge.

**Review by jusk02 — APPROVED** [review]

**Review by josemariogutierrez — APPROVED** [review]

**Review by camigomezdev — APPROVED** [review]
